modules/redis/src/main/scala/zio/redis/GenRedis.scala

```diff
@@ -0,0 +1,24 @@
+package zio.redis
+import zio.{IO, UIO}
+
+trait GenRedis[G[+_]]
```

Comment:
Wow. I thought this wasn't possible because after all this is the public interface of the library. But I've experimented a bit and it seems that as long as there's a public type alias for it, you can still use it just fine. TIL.